src/main/java/io/vertx/core/http/RequestOptions.java

```diff
@@ -333,7 +346,12 @@ private URL parseUrl(String surl) {
   public RequestOptions setAbsoluteURI(String absoluteURI) {
     Objects.requireNonNull(absoluteURI, "Cannot set a null absolute URI");
     URL url = parseUrl(absoluteURI);
-    Boolean ssl = false;
+    apply(url);
+    return this;
+  }
+
+  private void apply(URL url) {
```

Comment:
I think that this could simply be in the body `setAbsoluteURI(URL absoluteURI)` instead of being an independent method.

Reply:
Done